components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -1154,13 +1153,12 @@ func (backend *ESClient) SetDailyLatestToFalseForControlIndex(ctx context.Contex
 //GetNodesDayLatestTrue Get the Nodes from past 90 days from now where day latest and daily latest are true
 func (backend *ESClient) GetNodesDayLatestTrue(ctx context.Context, time90daysAgo time.Time) (map[string]relaxting.NodesUpgradation, error) {
```

Comment:
this method is written in such a way that it fetches the data for past N days, which is good, but based on description and name of parameters(time90daysAgo) other developers think it fetches for last 90 days. Please change the variable name to generic and description should be proper.